web/modules/custom/do_base/src/Hook/PageAttachmentsHook.php

```
@@ -0,0 +1,44 @@
+ * Page attachments hooks for do_base module.
+ */
+final class PageAttachmentsHook {
+
+  /**
+   * Implements hook_page_attachments().
+   *
+   * Adds an RSS autodiscovery link on the blog listing and individual blog posts.
+   */
+  #[Hook('page_attachments')]
+  public function pageAttachments(array &$attachments): void {
+    $current_path = \Drupal::service('path.current')->getPath();
```

PHPMD (2.15.0)

[error] The variable $current_path is not named in camelCase. (undefined)

(CamelCaseVariableName)